tests/integration/linux_bridge_test.py

```diff
@@ -397,6 +397,111 @@ def test_port_vlan_not_implemented(port0_up):
+    bridge_name = TEST_BRIDGE0
+
+    @pytest.mark.parametrize(
+        'is_native_vlan', [True, False], ids=['native', 'not-native']
+    )
+    def test_trunk_port_config(self, is_native_vlan, port0_up):
+        trunk_tags = self._generate_vlan_id_config(100, 101)
+        vlan_trunk_port_state = self._generate_vlan_filtering_config(
+            'trunk',
+            access_tag=self.access_tag if is_native_vlan else None,
+            trunk_tags=trunk_tags,
+            native_vlan=is_native_vlan,
+        )
+
+        port_name = port0_up[Interface.KEY][0][Interface.NAME]
+        bridge_state = _create_bridge_subtree_config((port_name,))
+        bridge_state[LinuxBridge.PORT_SUBTREE][0].update(vlan_trunk_port_state)
+        with linux_bridge(self.bridge_name, bridge_state) as desired_state:
+            assertlib.assert_state(desired_state)
+
+    @pytest.mark.parametrize(
+        'is_native_vlan', [True, False], ids=['native', 'not-native']
+    )
+    def test_multiple_trunk_ports_config(self, is_native_vlan, port0_up):
+        trunk_tags = self._generate_vlan_id_config(100, 102)
+        trunk_tags.append(self._generate_vlan_id_range_config(200, 299))
+        vlan_trunk_port_state = self._generate_vlan_filtering_config(
+            'trunk',
+            access_tag=self.access_tag if is_native_vlan else None,
+            trunk_tags=trunk_tags,
+            native_vlan=is_native_vlan,
+        )
+
+        port_name = port0_up[Interface.KEY][0][Interface.NAME]
+        bridge_state = _create_bridge_subtree_config((port_name,))
+        bridge_state[LinuxBridge.PORT_SUBTREE][0].update(vlan_trunk_port_state)
+        with linux_bridge(self.bridge_name, bridge_state) as desired_state:
+            assertlib.assert_state(desired_state)
+
+    def test_access_port_config(self, bridge0_with_port0, port0_up):
+        vlan_access_port_state = self._generate_vlan_filtering_config(
+            'access', access_tag=self.access_tag
+        )
+        port_name = port0_up[Interface.KEY][0][Interface.NAME]
+        bridge_state = _create_bridge_subtree_config((port_name,))
+        bridge_state[LinuxBridge.PORT_SUBTREE][0].update(
+            vlan_access_port_state
+        )
+        with linux_bridge(self.bridge_name, bridge_state) as desired_state:
+            assertlib.assert_state(desired_state)
+
+    @staticmethod
+    def _set_bridge_ports(bridge_state, *new_ports):
+        bridge_state[LinuxBridge.CONFIG_SUBTREE][LinuxBridge.PORT_SUBTREE] = [
+            port[Interface.KEY][0] for port in new_ports
+        ]
+
+    @staticmethod
+    def _get_port_by_name(linux_bridge, port_name):
+        for port in linux_bridge[LinuxBridge.CONFIG_SUBTREE][
+            LinuxBridge.PORT_SUBTREE
+        ]:
+            if port_name == port[LinuxBridge.PORT_NAME]:
+                return port
+
+    @staticmethod
+    def _generate_vlan_filtering_config(
+        port_type, trunk_tags=None, access_tag=None, native_vlan=None
```

Comment:
`access_tag` in the schema is named `tag`, right? If so, it should be the same here.

Reply:
Done.